src/app/shared/services/account/account.service.ts

```diff
@@ -201,9 +221,7 @@ export class AccountService {
 
   public async hasOwnArchives() {
     const archives = await this.refreshArchives();
-    const ownArchives = archives.filter(
-      (archive) => !archive.isPending()
-    );
+    const ownArchives = archives.filter((archive) => !archive.isPending());
```

Comment:
I know you only changed the formatting here and I haven't followed the whole code path, but it seems like we should not return "hasOwnArchives" true if the user only has deleted archives.

Reply:
(i.e., maybe we should check for archives in a good state instead of specifically excluding pending archives?)

Reply:
(But obviously that shouldn't be part of this PR!)